.editorconfig

```diff
@@ -2,13 +2,12 @@ root = true
 
 [*]
 charset = utf-8
-indent_size = 2
+indent_size = 4
```

Comment:
This also impacts yml or other files, we should leave them as indent_size for 2.